cpp/lib/include/VuFind.h

```diff
@@ -22,10 +22,7 @@
 
 #include <memory>
 #include <string>
-
-
-// Forward declaration:
-class DbConnection;
+#include "DbConnection.h"
```

Comment:
Why?

Reply:
Because if you have a script which includes only VuFind.h and tries to call GetDbConnection, this will fail if you don't include DbConnection.h in VuFind.h